packages/sitecore-jss/src/feaas/themes.ts

```diff
@@ -21,17 +21,19 @@ type RevisionType = 'staged' | 'published';
  */
 const FEAAS_LIBRARY_ID_REGEX = /-library--([^\s]+)/;
 
-export const FEAAS_SERVER_URL = 'https://feaas.blob.core.windows.net';
+export const FEAAS_SERVER_URL_STAGING = 'https://feaasstaging.blob.core.windows.net';
+export const FEAAS_SERVER_URL_BETA = 'https://feaasbeta.blob.core.windows.net';
+export const FEAAS_SERVER_URL_PROD = 'https://feaas.blob.core.windows.net';
 
 /**
  * Walks through rendering tree and returns list of links of all FEAAS Component Library Stylesheets that are used
  * @param {LayoutServiceData} layoutData Layout service data
- * @param {string} [serverUrl] server URL, default is @see {FEAAS_SERVER_URL} url
+ * @param {string} [sitecoreEdgeUrl] Sitecore Edge Platform URL. Default is https://edge-platform.sitecorecloud.io
  * @returns {HTMLLink[]} library stylesheet links
  */
 export function getFEAASLibraryStylesheetLinks(
   layoutData: LayoutServiceData,
-  serverUrl?: string
+  sitecoreEdgeUrl = 'https://edge-platform.sitecorecloud.io'
```

Comment:
Should this default url be shared? Since we use it also in a different place, or may be should be defined and imported from this place: https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss/src/graphql/graphql-edge-proxy.ts#L9